### PR TITLE
NO-JIRA: Add docs for cns volume migration

### PR DESCRIPTION
**Pull request by gnufied**

### CODE REVIEW

File: docs/cns-migration.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+```oc get pv -o json|jq -r '.items[] | select(.spec.csi.driver=="csi.vsphere.vmware.com")| .metadata.name'```
+
+
+### Now all PVs in the aforementioned file can be migrated via:
+
+```
+export KUBECONFIG=<path-to-kube-config>
+
+./cns-migration --source <source_datastore> --destination <destination_datastore> -volume-file <path_to_file_with_pv_names>```
```
Comment (dobsonj) [inline docs/cns-migration.md:45]: The ending ``` needs to be on a new line, otherwise the preformatted block lasts until EOF.

Comment (gnufied) [reply docs/cns-migration.md]: fixed!

**Pull request merged**